content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-terraform.md

```diff
@@ -29,131 +29,67 @@ This range of techniques helps to either temporarily or permanently use Pulumi a
 
 ## Referencing Terraform State
 
-Let's say your team already has some infrastructure stood up with Terraform. Maybe now isn't the time to convert it or maybe some part of your team wants to keep using Terraform for awhile, while you start adopting Pulumi. Often you'll want to interact with that infrastructure, maybe because it exports important IDs, IP addresses, configuration information, and so on. For example, it might define a VPC and you need its ID to create some new VMs in your new Pulumi project; or it may provision a Kubernetes cluster and you need the `kubeconfig` to deploy some application services into the cluster; etc.
+Pulumi allows you to reference output values from existing Terraform state files, enabling you to build new infrastructure that depends on resources provisioned with Terraform. This capability is particularly useful for:
 
-In each of these cases, you can use the `RemoteStateReference` resource to reference output variables exported from the Terraform project. This works for manually managed state files in addition to Terraform Cloud or Enterprise ones.
+* Organizations with existing Terraform infrastructure where the cost of migration isn't justified
+* Teams transitioning gradually from Terraform to Pulumi
+* Scenarios where some infrastructure must remain in Terraform due to organizational constraints
+* Accessing shared infrastructure (like VPCs, networks, or databases) managed by other teams
```

Comment:
Very minor grammar nit, we shouldn't capitalize the bullet points unless they are sentences (which they are not).